variants/STM32H5xx/H503RBT/variant_NUCLEO_H503RB.h

```diff
@@ -0,0 +1,211 @@
+#define PH1                     48
+
+// Alternate pins number
+#define PA0_ALT1                (PA0  | ALT1)
+#define PA0_ALT2                (PA0  | ALT2)
+#define PA1_ALT1                (PA1  | ALT1)
+#define PA2_ALT1                (PA2  | ALT1)
+#define PA3_ALT1                (PA3  | ALT1)
+#define PA4_ALT1                (PA4  | ALT1)
+#define PA5_ALT1                (PA5  | ALT1)
+#define PA7_ALT1                (PA7  | ALT1)
+#define PA7_ALT2                (PA7  | ALT2)
+#define PA8_ALT1                (PA8  | ALT1)
+#define PA8_ALT2                (PA8  | ALT2)
+#define PA9_ALT1                (PA9  | ALT1)
+#define PA10_ALT1               (PA10 | ALT1)
+#define PA11_ALT1               (PA11 | ALT1)
+#define PA12_ALT1               (PA12 | ALT1)
+#define PA13_ALT1               (PA13 | ALT1)
+#define PA14_ALT1               (PA14 | ALT1)
+#define PA14_ALT2               (PA14 | ALT2)
+#define PA15_ALT1               (PA15 | ALT1)
+#define PB0_ALT1                (PB0  | ALT1)
+#define PB1_ALT1                (PB1  | ALT1)
+#define PB1_ALT2                (PB1  | ALT2)
+#define PB2_ALT1                (PB2  | ALT1)
+#define PB3_ALT1                (PB3  | ALT1)
+#define PB4_ALT1                (PB4  | ALT1)
+#define PB4_ALT2                (PB4  | ALT2)
+#define PB5_ALT1                (PB5  | ALT1)
+#define PB6_ALT1                (PB6  | ALT1)
+#define PB6_ALT2                (PB6  | ALT2)
+#define PB7_ALT1                (PB7  | ALT1)
+#define PB13_ALT1               (PB13 | ALT1)
+#define PB14_ALT1               (PB14 | ALT1)
+#define PB15_ALT1               (PB15 | ALT1)
+#define PB15_ALT2               (PB15 | ALT2)
+#define PC1_ALT1                (PC1  | ALT1)
+#define PC2_ALT1                (PC2  | ALT1)
+#define PC3_ALT1                (PC3  | ALT1)
+#define PC6_ALT1                (PC6  | ALT1)
+#define PC7_ALT1                (PC7  | ALT1)
+#define PC8_ALT1                (PC8  | ALT1)
+#define PC9_ALT1                (PC9  | ALT1)
+#define PC12_ALT1               (PC12 | ALT1)
+
+#define NUM_DIGITAL_PINS        49
+#define NUM_ANALOG_INPUTS       16
+
+// On-board LED pin number
+#ifndef LED_BUILTIN
+  #define LED_BUILTIN           PNUM_NOT_DEFINED
+#endif
+
+// On-board user button
+#ifndef USER_BTN
+  #define USER_BTN              PNUM_NOT_DEFINED
+#endif
+
+// SPI definitions
+#ifndef PIN_SPI_SS
+  #define PIN_SPI_SS            PA0
+#endif
+#ifndef PIN_SPI_SS1
+  #define PIN_SPI_SS1           PA4_ALT1
+#endif
+#ifndef PIN_SPI_SS2
+  #define PIN_SPI_SS2           PA15_ALT1
+#endif
+#ifndef PIN_SPI_SS3
+  #define PIN_SPI_SS3           PB10
+#endif
+#ifndef PIN_SPI_MOSI
+  #define PIN_SPI_MOSI          PA3
+#endif
+#ifndef PIN_SPI_MISO
+  #define PIN_SPI_MISO          PA2
+#endif
+#ifndef PIN_SPI_SCK
+  #define PIN_SPI_SCK           PA1
+#endif
+
+// I2C definitions
+#ifndef PIN_WIRE_SDA
+  #define PIN_WIRE_SDA          PB3
+#endif
+#ifndef PIN_WIRE_SCL
+  #define PIN_WIRE_SCL          PB3
+#endif
```

Comment:
When pin properly ordered no need to define this as by default they are mapped to D10-D13 (SPI) and D14/D15 (I2C)